### DISCUSSION TITLE
OTA Update fails

### DISCUSSION THREAD

**Discussion by wallenium** [tbnobody/OpenDTU#1299] [Q&A] [answered]
Hi there,
Try to update from 23.8.3 to newer version via web Ui, but it do lot progress and is stuck at 99%.

Any idea how to solve that issue? Used the soldering kit linked on this page.

**Comment by DPO99** [comment]
Did you tried a restart before update?

**Reply by wallenium** [reply, marked as answer]
Yes, I restarted several times. Used the correct image (opendtu-generic_esp32), but it did not work.
tried it today again, it worked. no idea why...

**Comment by Freesee-10** [comment]
Before updating, be sure to switch off WLAN 5-GHZ and only work via WLAN 2.4 GHZ.
